openpype/plugins/publish/extract_review_slate.py

```diff
@@ -89,18 +89,20 @@ def process(self, instance):
                         tags = stream.get("tags") or {}
                         input_timecode = tags.get("timecode") or ""
                         if "width" in stream and "height" in stream:
-                            input_width = int(stream.get("width"))
-                            input_height = int(stream.get("height"))
+                            input_width = stream.get("width")
+                            input_height = stream.get("height")
```

Comment:
This is actually only place where the `int` conversion should be done here (to check if it's not `0`).